From lieb@sea-troll.net  Fri Jul 31 09:10:39 2009
From: Jim Lieb <lieb@sea-troll.net>
Date: Thu, 30 Jul 2009 10:30:32 -0700
Subject: Staging: vt665x: Text janitor in prep for driver merge
To: Greg KH <greg@kroah.com>, Forest Bond <forest@alittletooquiet.net>, Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Message-ID: <200907301030.33412.lieb@sea-troll.net>


From: Jim Lieb <lieb@canonical.com>

These changes in vt6656 match changes in vt6655 to minimize text differences
prior to merging the two trees into a single driver source.

Signed-off-by: Jim Lieb <lieb@canonical.com>
Cc: Forest Bond <forest@alittletooquiet.net>
Cc: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/vt6656/80211mgr.c |   10 +---------
 drivers/staging/vt6656/baseband.c |    8 ++++----
 drivers/staging/vt6656/hostap.c   |    2 +-
 3 files changed, 6 insertions(+), 14 deletions(-)

--- a/drivers/staging/vt6656/80211mgr.c
+++ b/drivers/staging/vt6656/80211mgr.c
@@ -18,7 +18,7 @@
  *
  * File: 80211mgr.c
  *
- * Purpose: Handles the managment frame parsing functions
+ * Purpose: Handles the 802.11 managment support functions
  *
  * Author: Lyndon Chen
  *
@@ -746,7 +746,6 @@ vMgrDecodeProbeResponse(
     )
 {
     PWLAN_IE    pItem;
-  //  BYTE        byCheckEID = 0;
 
 
     pFrame->pHdr = (PUWLAN_80211HDR)pFrame->pBuf;
@@ -764,13 +763,6 @@ vMgrDecodeProbeResponse(
                        + WLAN_PROBERESP_OFF_SSID);
 
     while( ((PBYTE)pItem) < (pFrame->pBuf + pFrame->len) ) {
-
-	  //20080701-01,<Remark> by Mike Liu
-      //  if (pItem->byElementID < byCheckEID)
-        //    break;
-        //else
-            //byCheckEID = pItem->byElementID;
-
         switch (pItem->byElementID) {
             case WLAN_EID_SSID:
                 if (pFrame->pSSID == NULL)
--- a/drivers/staging/vt6656/baseband.c
+++ b/drivers/staging/vt6656/baseband.c
@@ -743,7 +743,7 @@ BBuGetFrameTime (
         } else {
             uPreamble = 192;
         }
-        uFrameTime = (cbFrameLength * 80) / uRate;
+        uFrameTime = (cbFrameLength * 80) / uRate;  //?????
         uTmp = (uFrameTime * uRate) / 80;
         if (cbFrameLength != uTmp) {
             uFrameTime ++;
@@ -752,16 +752,16 @@ BBuGetFrameTime (
         return (uPreamble + uFrameTime);
     }
     else {
-        uFrameTime = (cbFrameLength * 8 + 22) / uRate;
+        uFrameTime = (cbFrameLength * 8 + 22) / uRate;   //????????
         uTmp = ((uFrameTime * uRate) - 22) / 8;
         if(cbFrameLength != uTmp) {
             uFrameTime ++;
         }
-        uFrameTime = uFrameTime * 4;
+        uFrameTime = uFrameTime * 4;    //???????
         if(byPktType != PK_TYPE_11A) {
             uFrameTime += 6;
         }
-        return (20 + uFrameTime);
+        return (20 + uFrameTime); //??????
     }
 }
 
--- a/drivers/staging/vt6656/hostap.c
+++ b/drivers/staging/vt6656/hostap.c
@@ -113,7 +113,7 @@ static int hostap_enable_hostapd(PSDevic
     DBG_PRT(MSG_LEVEL_DEBUG, KERN_INFO "%s: Enabling hostapd mode\n", dev->name);
 
 	pDevice->apdev = (struct net_device *)kmalloc(sizeof(struct net_device), GFP_KERNEL);
-    if (pDevice->apdev == NULL)
+	if (pDevice->apdev == NULL)
 		return -ENOMEM;
 	memset(pDevice->apdev, 0, sizeof(struct net_device));
 
